stdlib/join/sort_merge_join.go

```diff
@@ -72,23 +87,25 @@ func (SortMergeJoinPredicateRule) Rewrite(ctx context.Context, n plan.Node) (pla
 
 	successors := predecessors[0].Successors()
 
-	columns := make([]string, len(spec.On))
+	leftColumns := make([]string, 0, len(spec.On))
 	for _, pair := range spec.On {
-		columns = append(columns, pair.Left)
+		leftColumns = append(leftColumns, pair.Left)
 	}
-	successors[0] = makeSortNode(predecessors[0], columns)
+	successors[0] = makeSortNode(predecessors[0], leftColumns, "sortLeftSideJoinInput")
 
 	successors = predecessors[1].Successors()
 
-	columns = make([]string, len(spec.On))
+	rightColumns := make([]string, 0, len(spec.On))
 	for _, pair := range spec.On {
-		columns = append(columns, pair.Right)
+		rightColumns = append(rightColumns, pair.Right)
 	}
-	successors[0] = makeSortNode(predecessors[1], columns)
+	successors[0] = makeSortNode(predecessors[1], rightColumns, "sortRightSideJoinInput")
```

Comment:
Note: there was a bug here where the `columns` array contained an empty string `""` for its initial elements. This was actually caught by `ValidatePhysicalPlan()` since the sort nodes did not appear to provide the correct order to the join. I will file a separate PR to fix this.